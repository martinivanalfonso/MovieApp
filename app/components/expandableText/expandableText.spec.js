import React from 'react'
import { shallow } from 'enzyme'
import ExpandableText from './expandableText'

describe('ExpandableText Component', () => {
  test('renders correctly', () => {
    const component = shallow(<ExpandableText text="test" />)
    expect(component).toMatchSnapshot()
  })
})
