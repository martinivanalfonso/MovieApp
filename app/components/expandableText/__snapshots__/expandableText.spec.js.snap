// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExpandableText Component renders correctly 1`] = `
<View>
  <Text
    style={
      Object {
        "color": "gray",
        "fontSize": 16,
        "marginBottom": 14,
      }
    }
  >
    test
  </Text>
</View>
`;
